Rubberduck.VBEEditor/Extensions/ComponentTypeExtensions.cs

```diff
@@ -10,12 +11,18 @@ public static class ComponentTypeExtensions
         public const string FormBinaryExtension = ".frx";
         public const string DocClassExtension = ".doccls";
 
+        //TODO: double check whether the guesses below are correct.
```

Comment:
These are correct. There's also `.dsr` for ActiveXDesigner and `.res` for ResFile (which weren't in my original set)